packages/graphql-migrations/src/connector/read.ts

```diff
@@ -83,20 +82,28 @@ class Reader {
       const columnComments = await getColumnComments(this.knex, tableName, this.schemaName)
       const columnInfo: { [key: string]: Knex.ColumnInfo } = await this.knex(tableName)
         .withSchema(this.schemaName)
-        .columnInfo() as any
+        .columnInfo() as any;
+      // Primary key
+      const primaries = await getPrimaryKey(this.knex, tableName, this.schemaName)
+      
       // eslint-disable-next-line no-restricted-syntax
       for (const key in columnInfo) {
         if (columnInfo[key]) {
           const columnName = this.getColumnName(key)
           if (!columnName) { continue }
-          const info = columnInfo[key]
-          const foreign = foreignKeys.find((k: any) => k.column === key)
+          const info = columnInfo[key];
+          const {args, type} = getTypeAlias(info.type, info.maxLength);
+          const foreign = foreignKeys.find((k: any) => k.column === key);
+          const isPrimaryKey  = primaries.some((primary:  {column: string, indexName: string }) => primary.column === columnName);
           const column: TableColumn = {
             name: columnName,
+            isPrimaryKey,
+            autoIncrementable: type === "increments", // TODO
```

Comment:
Knex has bigIncrements for bigint, which could be mapped by the user in `migrateDB` to override the default mapping increments.

`` `suggestion
            autoIncrementable: type === "increments" || type === "bigIncrements", // TODO
```

Reply:
Good catch.